### PR TITLE
Autoformat scripts too

### PR DESCRIPTION
**Pull request by DRMacIver**
As observed in #540, scripts are not currently being autoformatted. As an increasingly large number of our scripts are written in Python rather than bash (yay!) this seems like an unfortunate oversight. This PR fixes that oversight.

### CODE REVIEW

File: scripts/check-changelog.py [hunk 1]
```diff
@@ -32,6 +51,6 @@
     if not any(d in changelog for d in acceptable_dates):
         print((
             "The current date (%s) isn't mentioned in the changelog. "
```
Comment (froztbyte) [inline scripts/check-changelog.py]: nit: quotation style differences

File: scripts/header.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # This file is part of Hypothesis, which may be found at
 # https://github.com/HypothesisWorks/hypothesis-python
 #
-# Most of this work is copyright (C) 2013-{year} David R. MacIver
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
```
Comment (froztbyte) [inline scripts/header.py]: is this accidental or intentional?

Comment (DRMacIver) [inline scripts/header.py]: Accidental. Thanks.

File: scripts/check-changelog.py (hunk 1 above)
Comment (DRMacIver) [inline scripts/check-changelog.py]: I believe that's the pyformat behaviour when you have a single quote inside. I've changed it so we instead escape the ' and pyformat seems happy to accept that.

File: scripts/check-changelog.py [hunk 3]
```diff
@@ -1,20 +1,39 @@
 #!/usr/bin/env python
```
Comment (alexwlchan) [inline scripts/check-changelog.py]: I’m not used to seeing an empty line here, but I’m not too fussed.

File: scripts/check_encoding_header.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 # This file is part of Hypothesis, which may be found at
```
Comment (alexwlchan) [inline scripts/check_encoding_header.py]: No shebang for this script?

File: scripts/enforce_header.py [hunk 5]
```diff
@@ -1,27 +1,29 @@
-from datetime import datetime
+from __future__ import division, print_function, absolute_import
```
Comment (alexwlchan) [inline scripts/enforce_header.py]: Even though it’s not enforced, do we want the header here anyway?

File: scripts/enforce_header.py [hunk 6]
```diff
@@ -1,27 +1,29 @@
...
+HEADER_FILE = 'scripts/header.py'
 
 CURRENT_YEAR = datetime.utcnow().year
 
 HEADER_SOURCE = open(HEADER_FILE).read().strip().format(year=CURRENT_YEAR)
 
 
 def main():
-    rootdir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+    rootdir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```
Comment (alexwlchan) [inline scripts/enforce_header.py]: Slightly prefer `os.path.dirname()`.

**Comment by DRMacIver** [comment]
Man I am *completely* confused about what's going on in this diff. Why has format suddenly picked up a  bunch of changes that it previously ignored?

**Comment by DRMacIver** [comment]
I'm going to wait to merge this until all the build wrangling is done if people don't mind.

File: scripts/enforce_header.py (hunk 5 above)
Comment (DRMacIver) [inline scripts/enforce_header.py]: I'm actually not sure *why* it's not enforced. I think it's a good idea to have it though, yeah. Will fix.

**Pull request merged**